README-template.md

```diff
@@ -678,6 +678,14 @@ The API Supports a number of options to configure the read
      </td>
      <td>Read</td>
    </tr>
+   <tr valign="top">
+     <td><code>enableListInference</code>
+     </td>
+     <td>  Indicates whether to use schema inference specifically for Parquet LIST logical type.
```

Comment:
Do you want to add a link here to the Pantheon doc?

Reply:
Sure